cmakeProject/libraries.cmake

```diff
@@ -0,0 +1,77 @@
+cmake_minimum_required(VERSION 3.13)
+
+function (includeLibraries)
+include(FetchContent)
+
+#  Include these libraries in CMakeLists.txt
+# tcMenu
+# IoAbstraction
+# TaskManagerIO
+# tcUnicodeHelper
+# TcMenuLog
+# AdafruitGFXNativePort
+# SimpleCollections
+
+
+SET(TCMENU_BASE_GIT_REPO "https://github.com/mwinters-stuff")
+# SET(TCMENU_BASE_GIT_REPO "https://github.com/TcMenu/tcMenuLib.git")
```

Comment:
Please can you make this definable in the script, defaulting to `https://github.com/TcMenu`

Reply:
In fact I'll just merge it and fix it up.